### PR TITLE
Differentiate schedule_to_start/close timeouts

### PR DESCRIPTION
**Pull request by lorensr**
Updated comments to define "schedule" time

### CODE REVIEW

File: temporal/api/command/v1/message.proto [hunk 1]
```diff
@@ -48,20 +48,24 @@ message ScheduleActivityTaskCommandAttributes {
...
+    // be set: it's useful in specific scenarios like worker-specific task queues. This timeout is
+    // always non retryable, as all a retry would achieve is to put it back into the same queue.
+    // Defaults to `schedule_to_close_timeout` or workflow execution timeout if that is not
+    // specified. More info:
+    // https://docs.temporal.io/docs/content/what-is-a-schedule-to-start-timeout/
     //
-    // (-- api-linter: core::0140::prepositions=disabled
-    //     aip.dev/not-precedent: "to" is used to indicate interval. --)
+    // (-- api-linter: core::0140::prepositions=disabled aip.dev/not-precedent: "to" is used to
+    //     indicate interval. --)
```
Comment (Sushisource) [inline temporal/api/command/v1/message.proto:68]: Don't think you want to change the line breaks for these bits

File: temporal/api/command/v1/message.proto [hunk 2]
```diff
@@ -48,20 +48,24 @@ message ScheduleActivityTaskCommandAttributes {
     temporal.api.taskqueue.v1.TaskQueue task_queue = 4;
     temporal.api.common.v1.Header header = 5;
     temporal.api.common.v1.Payloads input = 6;
-    // Indicates how long the caller is willing to wait for activity completion. Limits how long
-    // retries will be attempted. Either this or `start_to_close_timeout` must be specified. When
-    // not specified, defaults to the workflow execution timeout.
+    // Indicates how long the caller is willing to wait for activity completion. The "schedule" time
+    // is when the activity is initially scheduled—not when the most recent retry is scheduled.
```
Comment (Sushisource) [inline temporal/api/command/v1/message.proto:52]: Slightly odd formatting

```suggestion
    // is when the activity is initially scheduled - not when the most recent retry is scheduled.
```

**Review by Sushisource — APPROVED** [review]

File: temporal/api/command/v1/message.proto (hunk 2 above)
Comment (lorensr) [reply temporal/api/command/v1/message.proto:52]: It's an em dash, which is proper grammar here, but perhaps lost if it's always viewed in monospace—is there a documentation site these comments are displayed in?

Comment (Sushisource) [reply temporal/api/command/v1/message.proto]: Nope, these will typically be rendered by an IDE or read directly

**Pull request merged**